builder/docs_builder.livecodescript

```diff
@@ -17,10 +17,12 @@ command docsBuilderProgressUpdate pProgress, pMessage
    builderLog "message", pMessage
 end docsBuilderProgressUpdate
 
-command errorShow pError
-   builderLog "error", pError
-   throw pError
-end errorShow
+command logError pError
+   local tContext
+   // get the context of the calling handler: <handler>:<linenumber>
+   put item 2 to 3 of line -2 of the executionContexts into tContext
```

Comment:
@livecodeian it is safer to reference these items from the rear (-2 to -1) in the event someone (inexplicably I agree) has a stack name or stackFile path that contains the itemDelimiter. It did at one stage cause an error in libURL if I recall.